apps/whale/__sanity__/WhaleSanityContainer.sanity.ts

```diff
@@ -0,0 +1,59 @@
+import { WhaleSanityContainer } from '@defichain/testcontainers'
+import waitForExpect from 'wait-for-expect'
+
+const whale = new WhaleSanityContainer()
+
+// TODO(kodemon/eli-lim):
+//   would it make sense to use WhaleApiClient for sanity tests,
+//   instead of raw http requests?
```

Comment:
IMO, we probably should test both raw http and via WhaleApiClient, as it'll allow us to sanity test WhaleApiClient code too